### ISSUE TITLE
MP4Dash - HLS encrypted CBCS errors with Dolby Vision mp4 source

### ISSUE THREAD

**Issue by riba101** [axiomatic-systems/Bento4#898]
Hi there,

I'm running into an issue when I create an fairplay encrypted hls stream (CBCS) with mp4dash when I include a Dolby Vision file that was created on AWS MediaConvert. I also create a Dash stream with Widevine and Playready with the same source files and the stream is encrypted (CENC) and packaged correctly.

The error that occurs is 
ERROR: failed to process the file (-10)
ERROR: float division by zero

**Comment by riba101** [comment]
Update: Testing with files generated by the Dolby Encoding Engine also failed to package with CBCS enabled

**Comment by riba101** [comment]
Found the issue.

I had to add the following code to Source/C++/Core/Ap4CommonEncryption.cpp at line 293 (the AP4_CencCbcsSubSampleEncrypter::AP4_CencCbcsSubSampleEncrypter block)

```
if (!hvcc)    hvcc = AP4_DYNAMIC_CAST(AP4_HvccAtom, stsd->FindChild("dvhe/hvcC"));
if (!hvcc)    hvcc = AP4_DYNAMIC_CAST(AP4_HvccAtom, stsd->FindChild("dvh1/hvcC"));
```

I will create a PR for this

**Comment by barbibulle** [comment]
Thanks for finding this out!

**Comment by riba101** [comment]
Merged in PR #902
